core/mutator/entry/mutator_test.go

```diff
@@ -140,7 +135,7 @@ func TestCheckMutation(t *testing.T) {
 				entry: &tpb.Entry{
 					Index:            key,
 					Commitment:       []byte{2},
-					Previous:         nil,
+					Previous:         []byte("inavlidhash"),
```

Comment:
Not sure if the invalid spelling of invalid is part of the joke? :-)

Reply:
nope -- just a typo :-)